apps/vaporgui/FlowSubtabs.cpp

```diff
@@ -1,6 +1,39 @@
 #include "FlowSubtabs.h"
 #include "vapor/DataMgrUtils.h"
 
+#include "VFrame.h"
+#include "VSpinBox.h"
+#include "VComboBox.h"
+#include "VCheckBox.h"
+#include "VLineItem.h"
+#include "VSliderEdit.h"
+#include "VLineEdit.h"
+#include "VFileSelector.h"
+#include "VGeometry2.h"
+#include "VPushButton.h"
+
+#include <QScrollArea>
+
+#define verbose     1
+
+#define UNSTEADY_STRING    "Pathlines"
```

Comment:
Unless all of these macros are going possibly need to be changed with conditional compilation it's better to use const and an anonymous namespace:

namespace {
  const string Pathline = "Pathlines";
}

They are type-checked and can be evaluated by the debugger.

Reply:
Interesting.  Thanks.  Fixed.